main/src/main/java/org/mobilitydata/gtfsvalidator/report/model/FeedMetadata.java

```diff
@@ -121,6 +123,18 @@ private void loadFeedInfo(GtfsTableContainer<GtfsFeedInfo> feedTable) {
     }
   }
 
+  private boolean hasRouteNamesComponent(GtfsFeedContainer feedContainer) {
+    var routeContainer = feedContainer.getTableForFilename(GtfsRoute.FILENAME);
+    if (routeContainer.isPresent()) {
+      GtfsRouteTableContainer routeTable = (GtfsRouteTableContainer) routeContainer.get();
+      if (routeTable.hasColumn(GtfsRoute.ROUTE_SHORT_NAME_FIELD_NAME)
+          && routeTable.hasColumn(GtfsRoute.ROUTE_LONG_NAME_FIELD_NAME))
+        return routeTable.getEntities().stream()
+            .anyMatch(route -> route.hasRouteShortName() && route.hasRouteLongName());
```

Comment:
It's unclear to me if this should be an AND or an OR. The issue is a bit ambiguous on the subject IMO:

> We consider this component present if both the fields route_short_name and route_long_name in routes.txt are present, and if they have at least one value defined.

Does that mean that there should be one value for each field, or one value for either one of the fields?
Logically it seems that if you have either a short name or a long name it should be accepted. But it should be clarified.